### ISSUE TITLE
Plugin legacy name certbot-dns-hetzner:dns-hetzner may be removed in a future version

### ISSUE THREAD

**Issue by robertoschwald** [ctrlaltcoop/certbot-dns-hetzner#8]
Since today, I receive this warning on all hosts using this plugin:

Plugin legacy name certbot-dns-hetzner:dns-hetzner may be removed in a future version. Please use dns-hetzner instead.

```
# certbot renew --force-renewal
Saving debug log to /var/log/letsencrypt/letsencrypt.log

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
Processing /etc/letsencrypt/renewal/xxx.yyy.zzz.conf
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
Plugin legacy name certbot-dns-hetzner:dns-hetzner may be removed in a future version. Please use dns-hetzner instead.
Plugins selected: Authenticator certbot-dns-hetzner:dns-hetzner, Installer apache
Renewing an existing certificate
Running deploy-hook command: systemctl reload httpd

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
new certificate deployed with reload of apache server; fullchain is
...
```

So what should I do to migrate? Please add the info to the README. Thanks.

certbot 1.7.0
certbot-dns-hetzner 1.0.4

**Comment by l0rn** [comment]
To be completely honest - I don't know yet... I get this warning too and will figure it out eventually

**Comment by robertoschwald** [comment]
You can omit the prefix completely to get rid if the warning.
Note: You also need to remove the prefix from the ini file.
Maybe worth to update the README?

```
certbot certonly \
--authenticator dns-hetzner \
--dns-hetzner-credentials /etc/sysconfig/certbot-hetzner.ini \
--dns-hetzner-propagation-seconds 120 \
--email hostmaster@example.com --agree-tos -d myhost.example.com
```

Ini file:
```
dns_hetzner_api_token = ....
```

**Comment by l0rn** [comment]
cool thx for the reply, will amend the readme

**Comment by l0rn** [comment]
fixed with #10

**Comment by tim-seoss** [comment]
From what I can remember, this will break on older versions of certbot (which I think requires the previous syntax)?  I could be wrong, but this might be worth checking?
